packages/block-library/src/navigation/edit/index.js

```diff
@@ -310,6 +328,42 @@ function Navigation( {
 	] = useState();
 	const [ detectedOverlayColor, setDetectedOverlayColor ] = useState();
 
+	const [
+		showClassicMenuConversionErrorNotice,
+		hideClassicMenuConversionErrorNotice,
+	] = useNavigationNotice( {
+		name: 'block-library/core/navigation/classic-menu-conversion/error',
+	} );
+
+	function handleUpdateMenu( menuId ) {
+		setRef( menuId );
+		selectBlock( clientId );
+	}
+
+	useEffect( () => {
+		if ( classicMenuConversionStatus === 'pending' ) {
+			speak( __( 'Classic menu importing.' ) );
```

Comment:
These messages are pretty helpful, should we also add them as snackbar notifications? (In a different PR)

Reply:
We could yes if that's not too "noisy"? I wonder if notification popups have `speak()` built in?

Reply:
Actually I've found a great case where that would be very useful. When creating an empty menu it's not clear that it's been "successful". An explicit confirmation would be really helpful.

I'm working on a sister PR now which I will push up which refactors the create empty process.